### ISSUE TITLE
What to do next

### ISSUE THREAD

**Issue by CowNowK** [notgoodusename/OsirisAndExtra#593]
CSGO forced update to CS2, Osiris is useless at the moment

**Comment by notgoodusename** [comment]
Yep, sooooo, imma make a p2c in the near future, so you probably wont see any cs2 cheat source, unless you guys reaaaaalllly want the cheat

**Comment by Xsintashi** [comment]
> Yep, sooooo, imma make a p2c in the near future, so you probably wont see any cs2 cheat source, unless you guys reaaaaalllly want the cheat

Imma use my own external i been working on quite long. I used OsirisAndExtra project to c+p some things into my internal but stopped developing it as soon as i got red tf and OW banned. Pulled some code here as a recompesation for pasting. Anyway I very often check ur github and am open to see more projects from you

**Comment by notgoodusename** [comment]
> Imma use my own external i been working on quite long. I used OsirisAndExtra project to c+p some things into my internal but stopped developing it as soon as i got red tf and OW banned. Pulled some code here as a recompesation for pasting. Anyway I very often check ur github and am open to see more projects from you 

Thanks for your support 
I do have a private tf2 uncompleted cheat, might make it public
and regarding cs2, im not sure the purpose of making a public internal cheat because of the new anticheat, and active devs, so going forward you wont see many public cs2 cheats , most will probably be private/p2c

**Comment by notgoodusename** [comment]
Maybe I'll make a cheat that, even though it's detected, is open source for c+p code

**Comment by CowNowK** [comment]
cs2 added convars check, u can write memory but you'll get banned if u change convars(like InvChanger)🤔Develop a cs2 cheat maybe more difficult. Now I am trying develop my own cs2 cheat(I really need it🤡)
